server/src/main/kotlin/com/vishnurajeevan/libroabs/Application.kt

```diff
@@ -65,9 +67,26 @@ class Run : CliktCommand("run") {
     .flag(default = false)
 
   private val format: BookFormat by option("--format", envvar = "FORMAT")
-    .enum<BookFormat>()
+    .enum<BookFormat>(ignoreCase = true)
     .default(BookFormat.MP3)
 
+  private val parallelCount by option("--parallel-count", envvar = "PARALLEL_COUNT")
+    .int()
+    .restrictTo(
+      min = 1,
+      max = 3,
```

Comment:
in my testing, anything over 3 caused OOMs

Reply:
I wouldn't restrict it that much. You can easily increase memory at runtime if needed. 
But maybe we should mention this in the README?

Reply:
Yeah, I'm trying to find a balance between quick and easy and full end user control... Get ahead of issues like 

"i set my parallel count to 11 and the service is crashing with an out of memory" 

"did you check the readme?"

😅 


I've got 54 books and the service ran pretty quick on my end and its really only a useful thing for initial download anyways, once its running i cant imagine more than a few books a week getting pulled down

Reply:
ok. good point 😂
let's go with that and if it becomes too limiting we can always change it later